erpnext/controllers/accounts_controller.py

```diff
@@ -2349,7 +2347,6 @@ def set_payment_schedule(self):
 					)
 		else:
 			self.fetch_payment_terms_from_order(po_or_so, doctype)
-			self.ignore_default_payment_terms_template = 1
```

Comment:
This was added to address partial invoice scenario https://github.com/frappe/erpnext/pull/35285